src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.js

```diff
@@ -39,70 +49,56 @@ function BaseAutoCompleteSuggestions(props) {
+                onMouseDown={(e) => e.preventDefault()}
+                onPress={() => onSelect(index)}
+                onLongPress={() => {}}
+                accessibilityLabel={accessibilityLabelExtractor(item, index)}
+            >
+                {renderSuggestionMenuItem(item, index)}
+            </PressableWithFeedback>
+        ),
+        [highlightedSuggestionIndex, renderSuggestionMenuItem, onSelect, accessibilityLabelExtractor],
     );
 
-    /**
-     * This function is used to compute the layout of any given item in our list. Since we know that each item will have the exact same height, this is a performance optimization
-     * so that the heights can be determined before the options are rendered. Otherwise, the heights are determined when each option is rendering and it causes a lot of overhead on large
-     * lists.
-     *
-     * Also, `scrollToIndex` should be used in conjunction with `getItemLayout`, otherwise there is no way to know the location of offscreen indices or handle failures.
-     *
-     * @param {Array} data - This is the same as the data we pass into the component
-     * @param {Number} index the current item's index in the set of data
-     *
-     * @returns {Object}
-     */
-    const getItemLayout = (data, index) => ({
-        length: CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT,
-        offset: index * CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT,
-        index,
-    });
-
-    const innerHeight = CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT * props.suggestions.length;
+    const innerHeight = CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT * suggestions.length;
     const animatedStyles = useAnimatedStyle(() => StyleUtils.getAutoCompleteSuggestionContainerStyle(rowHeight.value));
 
     useEffect(() => {
-        rowHeight.value = withTiming(measureHeightOfSuggestionRows(props.suggestions.length, props.isSuggestionPickerLarge), {
+        rowHeight.value = withTiming(measureHeightOfSuggestionRows(suggestions.length, isSuggestionPickerLarge), {
             duration: 100,
             easing: Easing.inOut(Easing.ease),
         });
-    }, [props.suggestions.length, props.isSuggestionPickerLarge, rowHeight]);
+    }, [suggestions.length, isSuggestionPickerLarge, rowHeight]);
 
     useEffect(() => {
         if (!scrollRef.current) {
             return;
         }
-        scrollRef.current.scrollToIndex({index: props.highlightedSuggestionIndex, animated: true});
-    }, [props.highlightedSuggestionIndex]);
+        scrollRef.current.scrollToIndex({index: highlightedSuggestionIndex, animated: true});
+    }, [highlightedSuggestionIndex]);
 
     return (
         <Animated.View
-            ref={props.forwardedRef}
+            ref={forwardedRef}
             style={[styles.autoCompleteSuggestionsContainer, animatedStyles]}
             exiting={FadeOutDown.duration(100).easing(Easing.inOut(Easing.ease))}
         >
-            <FlatList
+            <FlashList
```

Comment:
We missed to add `estimatedListSize` which later caused https://github.com/Expensify/App/issues/32401